realsense2_camera/src/profile_manager.cpp

```diff
@@ -139,6 +139,43 @@ std::map<stream_index_pair, rs2::stream_profile> ProfilesManager::getDefaultProf
     return sip_default_profiles;
 }
 
+rs2::stream_profile VideoProfilesManager::validateAndGetSuitableProfile(rs2_stream stream_type, rs2::stream_profile given_profile)
+{
+    if (_all_profiles.empty()) 
+        throw std::runtime_error("Wrong commands sequence. No profiles set.");
```

Comment:
can you elaborate on this situation? can you please give an example ?

Reply:
This error check was already there in `getDefaultProfiles()` function. Will remove it as it is not required here.